modules/dnn/src/ie_ngraph.cpp

```diff
@@ -440,9 +448,27 @@ void InfEngineNgraphNet::init(Target targetId)
             ngraph_function->validate_nodes_and_infer_types();
         }
         cnn = InferenceEngine::CNNNetwork(ngraph_function);
-#ifdef _DEBUG  // TODO
-        //cnn.serialize("/tmp/cnn.xml", "/tmp/cnn.bin");
+
+        if (DNN_IE_SERIALIZE)
+        {
+#ifndef OPENCV_DNN_DISABLE_NETWORK_AUTO_DUMP
```

Comment:
Where this is defined? And what is default value?

Reply:
It is not defined anywhere in OpenCV, but can be passed through build flags.
Purpose is custom builds which want to turn off network dumps.

Reply:
I just though that `AUTO_DUMP` means to dump networks by default, without flags. If it's build flag, that means that if `OPENCV_DNN_DISABLE_NETWORK_AUTO_DUMP` is not specified but `OPENCV_DNN_IE_SERIALIZE` is ON we cannot dump the network?

Reply:
Naming is aligned with [this code](https://github.com/opencv/opencv/blame/master/modules/dnn/src/dnn.cpp#L3416-L3428).

Idea is compile-time blocker to dump/serialize network.

Reply:
So, basically my motivation is to have this option available in OpenCV from OpenVINO distribution. Sometimes to enable models in OpenVINO it's helpful to have this method. Will this build flag be available in OpenVINO build by default or we can miss it?

Reply:
This code is enabled by default, including OpenVINO builds.

Reply:
Oh, I see now, sorry. `#ifndef` and `DISABLE`. Thanks!